src/init/features/dataconnect/create_app.ts

```diff
@@ -0,0 +1,57 @@
+import { spawn } from "child_process";
+import * as clc from "colorette";
+import { logLabeledBullet } from "../../../utils";
+
+/** Create a React app using vite react template. */
+export async function createReactApp(webAppId: string): Promise<void> {
+  const args = ["create", "vite@latest", webAppId, "--", "--template", "react"];
+  await executeCommand("npm", args);
+}
+
+/** Create a Next.js app using create-next-app. */
+export async function createNextApp(webAppId: string): Promise<void> {
+  const args = [
+    "create-next-app@latest",
+    webAppId,
+    "--empty",
```

Comment:
I actually think we should leave this out. I know I was the one who suggested this...

Reply:
You mean the `--empty` flag, right?